lib/ClangImporter/ClangImporter.cpp

```diff
@@ -541,6 +541,25 @@ getNormalInvocationArguments(std::vector<std::string> &invocationArgStrs,
       });
     }
 
+    if (triple.isOSWindows()) {
+      switch (triple.getArch()) {
```

Comment:
No Itanium? :D

Reply:
SR-9999 Support Itanium

Reply:
Well, I did port my Linux distro to itanium ... 🤣